### ISSUE TITLE
Motor Unit Number Estimation Abnormality

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5936]
**Preferred term label:** Motor Unit Number Estimation Abnormality

**Synonyms** MUNE Abnormality

**Definition (free text, please give PubMed ID)** Abnormally high or low motor unit number estimation (-Pk Amps), PMID:28552243

**Parent term (use hpo.jax.org/app)** EMG abnormality HP:0003457

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I am not sure that this is a good HPO term, because it is hard to apply this method to an individual and say whether the result is abnormal or not -- i.e., there are no normal ranges. AFAIS the technique has been used in cohort studies and often shows a mean reduced value for cohorts with diseases such as SMA. 
To be a good HPO term, we would need to be able to say that a specific value in isolation is abnormal (possible with age/sex related norms).  Otherwise, perhaps there is a case for a term such as Declining motor unit number.

see for instance this text from (https://www.sciencedirect.com/topics/medicine-and-dentistry/motor-unit-number-estimation).
MUNE is not as useful as traditional EMG and
NCS in clinical diagnosis and management, princi-
pally because the wide range of MUNE values in
the healthy general population makes it difficult to
establish valid lower limits of normal for a diagnos-
tic MUNE study in individual patients. MUNE does
provide adjunctive information regarding rates of
motor unit decline and collateral reinnervation,
but there is no evidence yet that MUNE data sig-
nificantly improve diagnosis or clinical manage-
ment. Consequently, MUNE has not been
employed in routine clinical practice, and efforts
have been and remain focused on MUNE as a
research tool, as well as on the continual develop-
ment and refinement of MUNE techniques.
